### ISSUE TITLE
Cells get unexpected formatting when the parent worksheet is copied to a new workbook, while ok when copied inside workbook

### ISSUE THREAD

**Issue by dhnc4u** [ClosedXML/ClosedXML#582]
## Read and complete the full issue template

**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature

**Version of ClosedXML**

0.90

**What is the current behavior?**

Cells get unexpected formatting (borders appear in the attached sample, I've seen wrong background colors in real live files) when the parent worksheet is copied to another workbook. This behaviour is not seen when the sheet is copied inside its workbook.

**What is the expected behavior or new feature?**

The cell formatting should be kept. 
As far as I can tell there is no conditional formatting, nor specific cell styles, that would cause this behaviour.

**Did this work in previous versions of our tool?  Which versions?**

I don't know.

## Reproducibility

**Code to reproduce problem:**
```
Dim sInputFile As String = Path.Combine(Application.StartupPath, "xlsxfiles", "issue5.xlsx")
        Dim theWorkbook = New Excel.XLWorkbook(sInputFile)
        Dim newWorkbook = New Excel.XLWorkbook

        theWorkbook.Worksheet("dummy").CopyTo("copy")
        theWorkbook.Worksheet("dummy").CopyTo(newWorkbook, "copy of dummy")

        theWorkbook.SaveAs(Path.Combine(Path.GetDirectoryName(sInputFile), Path.GetFileNameWithoutExtension(sInputFile) & "_copyinside.xlsx"))
        newWorkbook.SaveAs(Path.Combine(Path.GetDirectoryName(sInputFile), Path.GetFileNameWithoutExtension(sInputFile) & "_copyoutside.xlsx"))
```

- [ x] I attached a sample spreadsheet.  (You can drag files on to this issue)
[issue5.xlsx](https://github.com/ClosedXML/ClosedXML/files/1443525/issue5.xlsx)

**Comment by igitur** [comment]
Cross workbook actions are still very much experimental.

**Comment by Pankraty** [comment]
It seems, this was fixed with #839
